backend/Cargo.toml

```diff
@@ -36,6 +36,12 @@ halo2_curves = { version = "0.6.1", package = "halo2curves", optional = true}
 snark-verifier = { git = "https://github.com/powdr-labs/snark-verifier", branch = "powdr-halo2", optional = true }
 halo2_solidity_verifier = { git = "https://github.com/powdr-labs/halo2-solidity-verifier", branch = "main", features = ["evm"], optional = true }
 
+p3-commit = { git = "https://github.com/plonky3/Plonky3.git", features = [
+    "test-utils",
+], optional = true }
+p3-matrix = { git = "https://github.com/plonky3/Plonky3.git", optional = true } 
+p3-uni-stark = { git = "https://github.com/plonky3/Plonky3.git", optional = true } 
```

Comment:
https://github.com/powdr-labs/powdr/pull/1917 changes the branch from main to specific commit, maybe you can already rebase on top of it?

Reply:
Ah it's merged! Will fix